datadotworld/models/dataset.py

```diff
@@ -115,6 +118,18 @@ def describe(self, resource=None):
         else:
             return self.__resources[resource].descriptor
 
+    @staticmethod
+    def _sanitize_resource(r):
```

Comment:
`cchardet` is incorrectly reading some utf-8-sig encoded files, including one of the ones used for unit tests, and so we're now explicitly providing the encoding. The data packages built by data.world should be including these though.